src/overlays/actors/ovl_Obj_Chan/z_obj_chan.c

```diff
@@ -150,8 +150,8 @@ void ObjChan_CalculatePotPosition(Vec3f* childPosOut, Vec3s* childRotOut, Vec3f*
     childRotOut->y += childAngle;
 }
 
-void ObjChan_InitChandelier(ObjChan* this, GlobalContext* globalCtx) {
-    s32 j;
+void ObjChan_InitChandelier(ObjChan* thisx, GlobalContext* globalCtx) {
+    ObjChan* this = thisx;
```

Comment:
Use `this2` when we need another `this`... although isn't this just going to be `Actor*`? The whole actor's pretty weird on this, actually, maybe every function takes `Actor*` and recasts?

Reply:
It does seem likely that most if not all the all functions take `Actor*` and do their own recasts. Kind of feels it would best in a second documentation pass. For now, I can just rename the `thisx` args in the relevant functions that have a recast.